configs/bokeh.json

```diff
@@ -1,7 +1,39 @@
 {
   "index_name": "bokeh",
-  "start_urls": [
-    "https://docs.bokeh.org"
+    "start_urls": [
+    {
+      "url": "https://docs.bokeh.org/en/(?P<version>.*?)/docs/user_guide.html",
+      "selectors_key": "user_guide",
+      "page_rank": 3
+    },
+    {
+      "url": "https://docs.bokeh.org/en/(?P<version>.*?)/docs/user_guide/",
```

Comment:
Having 2 times the same page might introduces duplicates

Reply:
It's unclear from the docs whether these are URLs to craw from or patterns to match against whenever happening to crawl any URL (or both)